packages/core/src/commands/cut.ts

```diff
@@ -0,0 +1,25 @@
+import { RawCommands } from '../types'
```

Comment:
heads up that this needs a .js ending now
```suggestion
import { RawCommands } from '../types.js'
```